src/utils/rest-api.js

```diff
@@ -486,12 +486,13 @@ export function fetchTwoWindingsTransformers(
     currentNodeUuid,
     substationsIds
 ) {
-    return fetchEquipments(
+    return fetchNetworkElementsInfos(
```

Comment:
I think we could improve readability by changing all "fetch{equipmen}" to "fetch{equipment}{infoType}" or by passing "infoType" as argument

Here we don't really know which info type we are fetching without taking a look at the function implementation

Reply:
yes but for all fetchGenerators, fetchBatteries, ... we don't know which infoType we are fetching